src/Visitation-Counters.ino

```diff
@@ -858,7 +873,17 @@ int setSolarMode(String command) // Function to force sending data in current ho
   else return 0;
 }
 
-int setSensorType(String command) // Function to force sending data in current hour
+/**
+ * @brief Set the Sensor Type object
+ * 
+ * @details Over time, we may want to develop and deploy other sensot types.  The idea of this code is to allow us to select the sensor
+ * we want via the console so all devices can run the same code.
+ * 
+ * @param Passes a string equal to "0" for pressure sensor and "1" for PIR sensor.  More sensor types possible in the future.
```

Comment:
I believe the proper syntax for doxygen here is `@param command Passes...`

Reply:
Done!  Thank you!